pygdf/column.py

```diff
@@ -56,8 +57,9 @@ def _concat(cls, objs):
             mask = Buffer.from_empty(mem)
             null_count = 0
             for o in objs:
-                mask.extend(o._get_mask_as_column().to_gpu_array())
-                null_count += o._null_count
+                if len(o) > 0:
```

Comment:
Since we previously filter out any objects that are empty, do we still need this check here?